plugins/render/api_key.go

```diff
@@ -0,0 +1,76 @@
+package render
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+type Config struct {
+	Version           int
+	SSHPreserveHosts  bool `yaml:"sshPreserveHosts"`
+	Profiles          map[string]Profile
+}
+
+type Profile struct {
+	DefaultRegion string `yaml:"defaultRegion"`
+	APIKey        string `yaml:"apiKey"`
+}
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://render.com/docs/api_key"),
+		ManagementURL: sdk.URL("https://console.render.com/user/security/tokens"),
```

Comment:
I'm not able to access either of these URLs, are these the correct ones?

Reply:
resolved this